tools/sort-i18n.py

```
@@ -128,37 +141,48 @@ def main():
 
             if args.write:
-                sorted_data = sort_translation_file(data, en_key_order)
+                sorted_data = sort_translation_file(data, en_key_order, args.prune_extra)
                 output_text = json.dumps(sorted_data, indent=4, ensure_ascii=False) + "\n"
                 with open(path, "w", encoding="utf-8", newline="\n") as f:
                     f.write(output_text)
-                print(f"    -> Rewritten with correct key order")
+                if needs_prune:
+                    print(f"    -> Rewritten with correct key order; pruned {len(extra_keys)} extra key(s)")
+                else:
+                    print(f"    -> Rewritten with correct key order")
```

Ruff (0.15.17)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)